go.mod

```diff
@@ -1,21 +1,23 @@
 module github.com/nutanix-cloud-native/cloud-provider-nutanix
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.1
 
 require (
 	github.com/google/uuid v1.5.0
 	github.com/nutanix-cloud-native/prism-go-client v0.4.0
-	github.com/onsi/ginkgo/v2 v2.15.0
-	github.com/onsi/gomega v1.31.1
-	k8s.io/api v0.29.1
-	k8s.io/apimachinery v0.29.1
-	k8s.io/client-go v0.29.1
+	github.com/onsi/ginkgo/v2 v2.17.1
+	github.com/onsi/gomega v1.32.0
+	k8s.io/api v0.30.1
+	k8s.io/apimachinery v0.30.1
+	k8s.io/client-go v0.30.1
 	k8s.io/cloud-provider v0.29.1
```

Comment:
You probably want this version to align with the other k8s.io updates as well

Reply:
Good catch!